airbrakes/imu/imu.py

```diff
@@ -101,19 +101,18 @@ def _fetch_data_loop(self, port: str, frequency: int, _: bool):
                         # This cpp file was the only place I was able to find all the channel names
                         # https://github.com/LORD-MicroStrain/MSCL/blob/master/MSCL/source/mscl/MicroStrain/MIP/MipTypes.cpp
                         # Check if the imu_data_packet has an attribute with the name of the channel
-                        if hasattr(imu_data_packet, channel):
+                        if hasattr(imu_data_packet, channel) or "Quaternion" in channel:
                             # First checks if the data point needs special handling, if not, just set the attribute
                             match channel:
                                 # These specific data points are matrix's rather than doubles
                                 case "estAttitudeUncertQuaternion" | "estOrientQuaternion":
                                     matrix = data_point.as_Matrix()
                                     # Converts the [4x1] matrix to the X, Y, Z, and W of the quaternion
```

Comment:
delete comment

Reply:
you mean delete this "Check if the imu_data_packet has an attribute with the name of the channel"?

Reply:
nah, I made the change I was talking about